src/cli/mod.rs

```diff
@@ -17,25 +17,41 @@
 
 use std::path::{Path, PathBuf};
 
-use clap::{Parser, Subcommand};
+use clap::Parser;
 
 use crate::app::config::get_data_dir;
 
 const LONG_ABOUT: &str = "
-Dft
+dft - DataFusion TUI
+
+CLI and terminal UI data analysis tool using Apache DataFusion as query
+execution engine.
+
+dft provides a rich terminal UI as well as a broad array of pre-integrated
+data sources, formats to make querying and analyzing data from
+various sources.
 
 Environment Variables
-RUST_LOG { trace | debug | info | error }: Standard across rust ecosystem for determining log level of application.  Default is info.
+RUST_LOG { trace | debug | info | error }: Standard rust logging level.  Default is info.
 ";
 
 #[derive(Clone, Debug, Parser)]
 #[command(author, version, about, long_about = LONG_ABOUT)]
 pub struct DftCli {
-    #[command(subcommand)]
-    pub command: Option<Command>,
+    #[clap(
+        short,
+        long,
+        num_args = 0..,
+        help = "Execute commands from file(s), then exit",
+        value_parser(parse_valid_file)
+    )]
+    pub file: Vec<PathBuf>,
```

Comment:
```suggestion
    pub files: Vec<PathBuf>,
```

Reply:
good call -- this was just copied from datafusion-cli. `files` is better